torch/_higher_order_ops/print.py

```diff
@@ -19,10 +20,30 @@ class Print(HigherOrderOperator):
     def __init__(self) -> None:
         super().__init__("print")
 
-    def __call__(self, format_str: str, **kwargs: object) -> object:
+    def __call__(self, format_str: str, **kwargs: object) -> None:
         assert isinstance(format_str, str)
         return super().__call__(format_str, **kwargs)
 
+    # pyrefly: ignore [bad-override]
+    def gen_schema(self, format_str: str, **kwargs: object) -> torch.FunctionSchema:
+        from torch._higher_order_ops.schema import HopSchemaGenerator
+
+        schema_gen = HopSchemaGenerator(self)
+        schema_gen.add_arg("format_str", format_str[0])
+
+        # Add each kwarg as a keyword-only argument
+        for key, value in kwargs.items():
+            # If the value is a list of bool, explicitly add each element as a separate argument
+            if isinstance(value, list) and all(isinstance(v, bool) for v in value):
+                for i, v in enumerate(value):
```

Comment:
no you shouldn't do this.. let's just not handle the case where the value is a list for now

Reply:
sounds good. I can quickly do another PR to support that in the schema in the future as well.